tests/requirements-relaxed.in

```diff
@@ -0,0 +1,13 @@
+# This requirements file containes no pins on antsibull-docs and other
+# essential tools
+# It's used for testing purposes and not for the production docs.ansible.com
+# builds.
+
+jinja2 >= 3.0.0 # used by hacking/build_library/build_ansible/command_plugins/generate_man.py and dump_keywords.py
+pyyaml >= 5.1 # used by ansible-core
+resolvelib >= 0.5.3, < 1.1.0 # used by ansible-core
+sphinx
+sphinx-notfound-page
+sphinx-ansible-theme
+rstcheck < 6 # # rstcheck 6.x has problem with rstcheck.core triggered by include files w/ sphinx directives https://github.com/rstcheck/rstcheck-core/issues/3
```

Comment:
Broken versions should still be in a separate file. It's possible to make an additional constraints file for this. If rstcheck is not a direct dependency of we don't rely on a public API that is missing from v6, it's not right to put it into the direct dependencies file and such a restriction should come from constraints.

Reply:
Yes, we do call rstcheck directly in https://github.com/ansible/ansible-documentation/blob/devel/tests/checkers/rstcheck.py. I'm not sure why we need a separate file for that. It's a direct dependency.

Reply:
But we don't depend on `< 6` features, we depend on any version. `< 6` happens to be a temporary constraint, not related to features we directly depend on. So semantically, it doesn't belong among the direct dependencies. A lower boundary would make sense if we knew since which version features we use appeared in the dependency, though.

Reply:
I don't think it's a good idea to add yet another constraints file, just so that we can remove all `<`, `>` from this file. The new file would contain half of the lines from this file (while this file won't get any shorter, only in terms of line length), and the packages mentioned in it still have to be mentioned in requirements-relaxed.in.

Reply:
I pushed https://github.com/ansible/ansible-documentation/pull/271/commits/577fbd81d51c74c5696f0cfbb1ee8e297e12c99b so we have something concrete to discuss, but yeah, I'm not convinced it's worthwhile to add yet another file. It may be more semantically correct, but it seems overly complicated to me.

Reply:
> just so that we can remove all `<`, `>` from this file

I disagree. This is not about adding or removing arbitrary bytes. This is about not making a mess of concepts being mixed up randomly with no meaning.